### ISSUE TITLE
Multi-block structures won't form the custom model

### ISSUE THREAD

**Issue by Geolite122** [BluSunrize/ImmersiveEngineering#3672]
So i was just playing casually with the mod right? But i wanted to make the assembler, when I try assemble it with the engineers hammer (as you would) it wouldn't form the custom model for the structure, and it wasn't just for the assembler, it seemed like all multi-block structures (well at least the tank, the silo, and the excavator). I thought it was a problem with a mod i had in the pack but when i deleted it it only fixed an issue with tinkers construct but not the multi-block structures issues, but that wasn't the only thing. The structure wouldn't be a working structure... So idk... I need help

**Comment by BluSunrize** [comment]
Please provide pictures of all angles, and all the different layers of the assembler
Also, please provide version infomation.

The issue template exists for a reason, please take a look at it, and improve your issue accordingly.

**Comment by Geolite122** [comment]
I am terrible sorry for not using the correct format, since this is my
first time using github for issues, but I must inform you that I wont be
able to send any screenshots since I am on a family trip and I did not
bring my pc, and I will not be back for a week or more. But I did test out
IE with no mods besides jei (1.12.2) and the problem is still attendant. I
am confuse what the problem could be.

Sincerely, Geo
